iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/voter/FittingOrgWordsMatchVoter.java

```diff
@@ -0,0 +1,130 @@
+package eu.dnetlib.iis.wf.affmatching.match.voter;
+
+import java.util.List;
+import java.util.Set;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Sets;
+
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+
+/**
+ * Match voter that checks if {@link AffMatchOrganization#getName()} words
+ * are present in {@link AffMatchAffiliation#getOrganizationName()} words.
+ * 
+ * @author madryk
+ */
+public class FittingOrgWordsMatchVoter implements AffOrgMatchVoter {
+
+    private static final long serialVersionUID = 1L;
+    
+    
+    private List<Character> charsToFilter;
+    
+    private double minFittingOrgWordsPercentage;
+    
+    private double minFittingWordSimilarity;
+    
+    private int minWordLength;
+    
+    
+    //------------------------ CONSTRUCTORS --------------------------
+    
+    /**
+     * Default constructor
+     * 
+     * @param charsToFilter - list of characters that will be filtered out before comparing words
+     * @param minWordLength - words with length equal or less than 
+     *      this value will be filtered out before comparing words.
+     *      Setting it to zero disables this feature.
+     * @param minFittingOrgWordsPercentage - percentage of {@link AffMatchOrganization#getName()}
+     *      words that have to be found in {@link AffMatchAffiliation#getOrganizationName()}.
+     *      Value must be between (0,1].
```

Comment:
Well, because it is not percentage then I suggest changing the name to minFittingOrgWordsRatio